allofplos/transformations.py

```diff
@@ -30,13 +30,16 @@
 
 def filename_to_url(filename, plos_network=False):
     """
-    For a local XML file in the corpusdir directory, transform it to the downloadable URL where its XML resides
+    For a local XML file in the get_corpus_dir() directory, transform it to the downloadable URL where its XML resides
     Includes transform for the 'annotation' DOIs
     Example:
     filename_to_url('allofplos_xml/journal.pone.1000001.xml') = \
     'http://journals.plos.org/plosone/article/file?id=10.1371/journal.pone.1000001'
-    :param file: relative path to local XML file in the corpusdir directory
-    :param directory: defaults to corpusdir, containing article files
+    
+    TODO: directory is neither a parameter that is given nor one that is used.
```

Comment:
ok to remove directory info for the 'filename' transformations which indeed do not depend on directory (those with 'path' do).

Reply:
they're gone